### PR TITLE
Add instructions for WebView2 fallack

### PR DESCRIPTION
**Pull request by kkrum**
Add instructions for how to fall back the registration experience to webview2

### CODE REVIEW

File: content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/_index.md [hunk 1]
```diff
@@ -56,6 +56,11 @@ The on-disk configuration of the Windows client can be changed at any time by mo
 
 Changes to this file are processed immediately by the WARP client.
 
+### Authentication Experience
+By default WARP will use the default browser associated with the user to performance registration. You can override this to optionally uses an embeded browser to work around protocol handler issues. To use an embeded browser you must complete the following steps:
```
Comment (naga-cf) [inline content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/_index.md]: optionally uses an -> optionally use an
embeded->embedded

Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/_index.md]: What are these protocol handler issues?

File: content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/_index.md [hunk 2]
```diff
@@ -56,6 +56,11 @@ The on-disk configuration of the Windows client can be changed at any time by mo
 
 Changes to this file are processed immediately by the WARP client.
 
+### Authentication Experience
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/_index.md]: Is the browser embedded in the WARP client or some other app?

File: content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/_index.md (hunk 1 above)
Comment (kkrum) [reply content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/_index.md]: The main one we are working around appears to be a bug in Chromium on Windows but there are lots of things that could cause issues with our protocol handler. Another example is Chromium has tech to detect if a protocol handler is being called too often (like if a junk ad was trying to take over the browser) and not launch the associated app.

File: content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/_index.md (hunk 2 above)
Comment (kkrum) [reply content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/_index.md]: The WebView2 embedded browser must be downloaded from Microsoft. Then our app launches that component as part of an in-app experience. 

Else we by default use the default. browser on the system

**Review by ranbel — APPROVED** [review]

**Pull request merged**